docs/user-guide/composer.md

```diff
@@ -1,3 +1,45 @@
-# Composer
 
-TODO
+ This guide will walk you through the process of using our email composition tool effectively.
+ ###  Creating a New Email in web
+1. Click on the "New Email" button.
+2. A new email composer will appear.
+3. From: By default it will display your default identity. When you click on `Edit` icon (3.1)- a drop-down list of your identities will be shown and you can select one of them
+4. Fill in the recipient's email address in the "To" field.
+5. Enter a subject for your email in the "Subject" field.
+6. Adding Recipients
+You can add multiple recipients by separating their email addresses with commas. Use the "CC" (Carbon Copy) and "BCC" (Blind Carbon Copy) fields for additional recipients as needed.
+- (6.1)`CC`: Carbon Copy:  Anyone in this field will see the other recipients of the message.
+- (6.2) `BCC`: Blind Carbon Copy: The recipients won't know that you added anyone to 'Bcc'.
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/7eb71fd0-0f2c-4b9b-9bbe-e5e55e36f596)
+
+7. Text Formatting
+Use the formatting options provided to style your email's text. These include :
+- (7.1) `Style`
+- (7.2) `Font famlily`
+- (7.3) `Text color`
+- (7.4) `Text background color`
+- (7.5) `Bold`
+- (7.6) `Italic`
+- (7.7) `Underline`
+- (7.8) `Strikethrough`
+- (7.9) `Paragraph`
+- (7.10) `Order list` 
+
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/e911b687-d1ba-49da-943b-1c6312043144)
+
+8. Adding Attachments
+To attach files, click on the `attachment icon` and select the files you want to include. 
+9. Insert images
+To insert images to the email body as an inline image, clock on `image icon` then select images that you want to include 
+10. Code view
+When you click on `Code view button`, the email your are writing is displayed in Code view mode
+11. Sending Your Email
+- Click the "Send" button to send your email. Once sent, it will appear in your "Sent" folder.
+- You can also click button `Save to Draft` (12) then the email is moved to Draft folder, you can then open it again and continue composing it.
+- If you click `Cancel button` (13), the email will be discarded 
+
+14. Managing Sent Emails
+To access sent emails or track the status of your sent email, navigate to the "Sent" folder within your email client.
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/dc30a422-0dc5-42bd-ab21-5b1d4a543a01)
+
+
```

Comment:
How about `read receipts` option? Please add one paragraph and maybe an image about it.